Leftpad.rb

```diff
@@ -1,7 +1,33 @@
-stringIn = gets.chomp
-timesIn = gets.to_i
-charIn = gets.chomp
+def left_pad(string = "You are extremely lazy, Kid", times = 1, char = " ")
+  puts "Choose a word:"
+  string_in = gets.chomp
+  puts "Choose a number:"
+  times_in = gets.to_i
+  puts "Choose a character:"
+  char_in = gets.chomp
+  puts "Would you like to have your word of choice before all the other characters? Y/N"
+  bool = gets.chomp
+  bool.downcase!
 
-chars = charIn * timesIn
+  if string_in == nil || string_in == ""
+    string_in = string
+  end
 
-print "Your characters&word coctail is: #{chars}#{stringIn}."
+  if times_in == nil || times_in == ""
+    times_in = times
+  end
+
+  if char_in == nil || char_in == ""
+    char_in = char
+  end
+
+  chars = char_in * times_in
+
+  if bool == "y"
+    print "Your word&characters coctail is: #{string_in}#{chars}."
```

Comment:
consider using `puts` instead of `print`. The newline character, which `puts` adds at the end of the line, makes for more user-friendly output

Reply:
I am wondering myself how it managed to get into the commit (how many other `print`s you've seen here, right?).  Thanks for pointing that out.